openjdk/src/test/java/org/conscrypt/PlatformTestUtil.java

```diff
@@ -0,0 +1,13 @@
+package org.conscrypt;
+
+import org.junit.Assume;
+
+final class PlatformTestUtil {
+    static boolean isJavaVersion(int version) {
+        return Platform.javaVersion() >= version;
+    }
+
+    static void assumeJava8() {
```

Comment:
Moving these out to their own class seems unnecessary, I'd leave them as helpers in PlatformTest or move them to TestUtils.

Reply:
I did not want to make ConscryptEngineTest depend on PlatformTest, tests should be independent.
Also TestUtils inside testing module does not see the org.conscrypt.Platform.javaVersion() method.
Can I leave it as is?

Reply:
TestUtils duplicates the javaversion logic from Platform a bit now, but I could live with that.